src/preferences/dialog/dlgprefwaveform.cpp

```diff
@@ -227,6 +236,12 @@ void DlgPrefWaveform::slotResetToDefaults() {
     playMarkerPositionSlider->setValue(50);
 }
 
+void DlgPrefWaveform::notifyRebootNecessary() {
+    // make the fact that you have to restart mixxx more obvious
+    QMessageBox::information(
+            this, tr("Information"), tr("Mixxx must be restarted to load the new overview type."));
```

Comment:
This is used both for changing the skin and overview types.

Reply:
?
the skin and locale are set in src/preferences/dialog/dlgprefinterface.cpp

Reply:
Read the text here. It only mentions the overview type.

Reply:
yes, read the file name please :)
we're in src/preferences/dialog/dlgpref**waveform**.cpp and here the only option that requires a skin reload is the overview.

Reply:
src/preferences/dialog/dlgpref**interface**.cpp has different messages

Reply:
Okay. Sorry for the noise.